cypress/e2e/signIn.cy.js

```diff
@@ -1,7 +1,31 @@
 /// <reference types="cypress" />
 
 describe('Sign In page', () => {
+  const randomNumber = Math.random().toString().slice(2, 5);
+  const username = `test${randomNumber}`;
+  const email = `${username}@gmail.com`;
+  const password = 'Testpass1!';
+
   it('should provide an ability to log in', () => {
+    cy.visit('https://conduit.mate.academy/');
+
+    cy.contains('Sign up').click();
+
+    cy.get(':nth-child(1) > .form-control').type(username);
+    cy.get(':nth-child(2) > .form-control').type(email);
+    cy.get(':nth-child(3) > .form-control').type(password);
```

Comment:
The selectors used to target the form inputs are not very descriptive. If the order of the form fields changes in the future, these tests will break. Instead, consider using data-testid attributes or other more descriptive selectors to target these elements. This will make your tests more robust and easier to understand.